README.md

```diff
@@ -96,17 +96,18 @@ evaluation = MTEB(task_categories=['S2S']) # Only select sentence2sentence datas
 * by their languages
 
 ```python
+import mteb
 evaluation = MTEB(task_langs=["en", "de"]) # Only select datasets which are "en", "de" or "en-de"
```

Comment:
If you want to add an import here, it should probably be `from mteb import MTEB`, no? But I don't think an import here is needed as it's very clear from the script under `## Usage` I think 🤔

Reply:
fixed